internal/commands/all/all.go

```diff
@@ -134,6 +135,11 @@ func BuildCommands(rootCmd *cobra.Command, conf *config.Config) {
 		commands.BuildCommand(databaseproperties.ShowCommand(i.serviceType, i.serviceName), typeCommand.Cobra(), conf)
 	}
 
```

Comment:
Could also add OpenSearch as subcommand into above properties command, if it has properties available

Reply:
Added